### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by AayushTyagiKD**
Homework 1 Code
Date: 04/01/2022 Session-1

### CODE REVIEW

File: Program2.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+public class Program2
+{
+    void modThree(int N)
+    {
+        for(int i=1; i<=N; i++)
```
Comment (parthbhope-kd) [inline Program2.java:6]: You had to check if the sum of digits is divisible by three

Comment (AayushTyagiKD) [reply Program2.java:6]: Okay fixing it!

**Review by parthbhope-kd — APPROVED** [review]

**Comment by parthbhope-kd** [comment]
Change the destination branch to homework

**Comment by AayushTyagiKD** [comment]
> Change the destination branch to homework

Updated the destination branch to homework.
Please Merge

**Pull request merged**